.github/workflows/main.yml

```diff
@@ -21,8 +21,13 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install -r requirements.txt
         python -m platformio update
+        python -m platformio platform install native
+        python -m platformio platform install teensy
 
-    - name: CXX software tests
+    - name: Desktop software tests
       run: |
-        python -m platformio platform install native
         python -m platformio test --verbose -e github
+        
+    - name: Teensy build
+      run: |
+        python -m platformio run --verbose -e teensy35
```

Comment:
Maybe add teensy36 as well.

Reply:
I'll go ahead and do that. And I'm going to make a separate compile target for teensy ci builds only to avoid the issue with testing on teensy.